packages/headings/src/sass/module.scss

```diff
@@ -0,0 +1,38 @@
+//--------------------------------------------------------------------------------------------------------------------------------------------------------------
+// Headings module
+// Class styles for display headings.
+//
+// Content:
+// - loop that generates .display-[1…6]
+// - modifications and additions to those .display-[1–6] classes
+//--------------------------------------------------------------------------------------------------------------------------------------------------------------
+
+
+@import 'globals.scss';
+
+
+//--------------------------------------------------------------------------------------------------------------------------------------------------------------
+// HEADING/DISPLAY STYLES
+//--------------------------------------------------------------------------------------------------------------------------------------------------------------
+/**
+ * Generates .display-[1…6] classes for display headings.
+ *
+ * The magic number here reduces the leading for these heading displays, since
+ * the body text leading is too large at heading/display sizes.
+ *
+ * We also set the font-size automatically too. (:
+ */
+@for $i from 1 through 6 {
+	.display-#{$i} {
+		line-height: $uikit-leading; //1.142857143;//1.4 from 16 (default: core)
+		margin-top: uikit-space( 1,1 ); //1 unit at the new line-height (1.4 -- 1.4em)
+		margin-bottom: uikit-space( .5,1 ); //1 unit at the new line-height (1.4 -- 0.7em)
```

Comment:
haha I meant between 0.5 and the comma so you can see that there are two values:
```scss
margin-bottom: uikit-space( .5,1 );
```
vs
```scss
margin-bottom: uikit-space( .5, 1 );
```

Not important :)